app/code/Magebit/Faq/Ui/Component/Listing/Column/QuestionActions.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * Magebit_Faq
+ *
+ * @category     Magebit
+ * @package      Magebit_Faq
+ * @author       Artjoms Fomins <info@magebit.com>
+ * @copyright    Copyright (c) 2023 Magebit, Ltd.(https://www.magebit.com/)
+ */
+
+declare(strict_types=1);
+
+namespace Magebit\Faq\Ui\Component\Listing\Column;
+
+use Magento\Framework\UrlInterface;
+use Magento\Framework\View\Element\UiComponent\ContextInterface;
+use Magento\Framework\View\Element\UiComponentFactory;
+use Magento\Ui\Component\Listing\Columns\Column;
+use Magento\Framework\App\Rss\UrlBuilder;
+
+/**
+ * Class prepare Question Actions
+ */
+class QuestionActions extends Column
+{
+    /** Url path */
+    public const CMS_URL_PATH_EDIT = 'faq/question/edit';
+    public const CMS_URL_PATH_DELETE = 'faq/question/delete';
+
+    protected UrlBuilder $actionUrlBuilder;
+    protected \Magento\Framework\UrlInterface $urlBuilder;
+
+    /**
+     * @param ContextInterface $context
+     * @param UiComponentFactory $uiComponentFactory
+     * @param UrlBuilder $actionUrlBuilder
+     * @param UrlInterface $urlBuilder
+     * @param array $components
+     * @param array $data
+     */
+    public function __construct(
+        ContextInterface $context,
+        UiComponentFactory $uiComponentFactory,
+        UrlBuilder $actionUrlBuilder,
+        UrlInterface $urlBuilder,
+        array $components = [],
+        array $data = []
+    ) {
+        $this->urlBuilder = $urlBuilder;
+        $this->actionUrlBuilder = $actionUrlBuilder;
+        parent::__construct($context, $uiComponentFactory, $components, $data);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function prepareDataSource(array $dataSource)
+    {
+        if (isset($dataSource['data']['items'])) {
+            $storeId = $this->context->getFilterParam('id');
+
+            foreach ($dataSource['data']['items'] as &$item) {
+                $item[$this->getData('name')]['edit'] = [
+                    'href' => $this->urlBuilder->getUrl(
+                        self::CMS_URL_PATH_EDIT,
+                        ['id' => $item['id'], 'store' => $storeId]
+                    ),
+                    'label' => __('Edit'),
+                    'hidden' => false,
+                ];
+                $item[$this->getData('name')]['delete'] = [
+                    'href' => $this->urlBuilder->getUrl(self::CMS_URL_PATH_DELETE, ['id' => $item['id']]),
+                    'label' => __('Delete'),
+                    'confirm' => [
+                        'title' => __('Delete %1', $item['question']),
+                        'message' => __('Are you sure you want to delete a %1 record?', $item['question']),
```

Comment:
As you are sanitizing the $item['question'] value in the frontend, you should do the same in the backend.